SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestSettings.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialTestSettings);
+
+FSpatialTestSettings::FSpatialTestSettings()
+	: OriginalSpatialGDKSettings(nullptr)
+	, OriginalLevelEditorPlaySettings(nullptr)
+	, OriginalSpatialGDKEditorSettings(nullptr)
+	, OriginalGeneralProjectSettings(nullptr)
+	, OriginalEditorPerformanceSettings(nullptr)
+{
+}
+
+const FString FSpatialTestSettings::OverrideSettingsFileExtension = TEXT(".ini");
+const FString FSpatialTestSettings::OverrideSettingsFileDirectoryName = TEXT("MapSettingsOverrides");
+const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides");
```

Comment:
Valentyn's comment from the other PR:
```suggestion
const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides_");
```

Reply:
would need an update of the TestGymPR